features/verifysession/impl/src/main/kotlin/io/element/android/features/verifysession/impl/VerifySelfSessionView.kt

```diff
@@ -239,21 +236,12 @@ private fun BottomMenu(
 
     when (verificationViewState) {
         is FlowStep.Initial -> {
-            if (verificationViewState.isLastDevice) {
-                BottomMenu(
-                    positiveButtonTitle = stringResource(R.string.screen_session_verification_enter_recovery_key),
-                    onPositiveButtonClicked = onEnterRecoveryKey,
-                    negativeButtonTitle = stringResource(R.string.screen_identity_confirmation_create_new_recovery_key),
-                    onNegativeButtonClicked = onCreateNewRecoveryKey,
-                )
-            } else {
-                BottomMenu(
-                    positiveButtonTitle = stringResource(R.string.screen_identity_use_another_device),
-                    onPositiveButtonClicked = { eventSink(VerifySelfSessionViewEvents.RequestVerification) },
-                    negativeButtonTitle = stringResource(R.string.screen_session_verification_enter_recovery_key),
-                    onNegativeButtonClicked = onEnterRecoveryKey,
-                )
-            }
+            BottomMenu(
+                positiveButtonTitle = stringResource(R.string.screen_identity_use_another_device),
```

Comment:
I do not think we should propose to `"Use another device"` if we know that this is the latest session (`isLastDevice`). Maybe the UX can be better here by just hiding the first button?

Reply:
Good call!